upgrade/1.2/scripts/upgrade/prerequisites.sh

```diff
@@ -252,8 +252,9 @@ state_name="UPDATE_DOC_RPM"
 state_recorded=$(is_state_recorded "${state_name}" $(hostname))
 if [[ $state_recorded == "0" ]]; then
     echo "====> ${state_name} ..."
-    if [[ ! -f docs-csm-latest.noarch.rpm ]]; then
-        echo "Please make sure 'docs-csm-latest.noarch.rpm' exists under: $(pwd)"
+    if [[ ! -f /root/docs-csm-latest.noarch.rpm ]]; then
```

Comment:
Is this supposed to be docs-csm-* like the cp command in upgrade/1.2/Stage_0_Prerequisites.md ?

Reply:
they should be the same thing since we tell them to download the 'latest' and that's what's being pulled from nexus